server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+
+A condition can be configurable the same as other configurable provider. What can be configured depends on each condition's nature.
+
+The following conditions are provided:
+
+The way of creating/updating a client::
+    * Dynamic Client Registration (Anonymous or Authenticated with Initial access token or Registration access token)
+    * Admin REST API (Admin Console, etc.)
+
+So for example when creating a client, a condition can be configured to evaluate to true when this client is created by OIDC Dynamic Client Registration without initial
+access token (Anonymous Dynamic Client Registration). So this condition can be used for example to ensure that all clients registered through OIDC Dynamic Client Registration
+are FAPI compliant.
+
+Author of a client (Checked by presence to the particular role or group)::
+    On OpenID Connect dynamic client registration, an author of a client is the end user who was authenticated to get an access token for generating a new client, not Service
+    Account of the existing client that actually accesses the registration endpoint with the access token. On registration by Admin REST API, an author of a client is the end user
+    like administrator of the {project_name}.
+
+Client Access Type (confidential, public, bearer-only)::
+    For example when a client sends an authorization request, a policy is adopted if this client is confidential.
+
+Client Scope::
+    Evaluates to true if client has particular client scope (either as default or as an optional scope used in current request). This can be used for example to ensure that
+    OIDC authorization requests with scope `fapi-example-scope` need to be FAPI compliant.
+
+Client Role::
+    Applies for clients with the client role of the specified name
+
+Client Domain Name, Host or IP Address::
+    Applied for specific domain names of client. Or for the cases when administrator registers/updates client from particular Host or IP Address.
+
+Any Client::
+    This condition always evaluates to true. It can be used for example to ensure that all clients in the particular realm are FAPI compliant.
+
+===== Executor
+
+An executor specifies what action is executed on a client to which a policy is adopted. The executor executes one or several specified actions. For example,
+some executor checks whether the value of the parameter `redirect_uri` in the authorization request matches exactly with one of the pre-registered redirect URIs on
+Authorization Endpoint and rejects this request if not.
+
+The executor can not be used solely by itself. It can be used in a <<_client_policy_profile,profile>> that is described afterwards.
+
+An executor has it's own SPI. Developers can implement their own executor by implementing the particular provider, however SPI is considered private and can change in the future.
```

Comment:
Should we even mention the SPI if it's private?

Reply:
Removing (See above).